Popularity/Kipras/pyspark/rollOnSpark.sh

```diff
@@ -0,0 +1,249 @@
+#!/bin/bash
```

Comment:
I believe I had this question before. I asked whether I have to rewrite all the merge, transform ect. functions on spark and you said that we only want to perform classification part on spark. So how is it ?

Or maybe I don't understand something and you mean that we can run existing transform merge ect scripts on data that is on HDFS?

Please clarify a bit this part.